packages/certs-v5/lib/endpoints.js

```diff
@@ -61,32 +45,9 @@ async function all(appName, heroku) {
   const featureList = await heroku.get(`/apps/${appName}/features`)
   const privateSniFeatureEnabled = checkPrivateSniFeature(featureList)
```

Comment:
all spaces are using private space sni now, so we don't need this lookup

Reply:
cool, removed at 4f05dc2d7